From ae4ca4dca4f81b0da94919851503d3401f3b40f0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mika=20Bostr=C3=B6m?= <bostik@odin.bostik.iki.fi>
Date: Fri, 16 Dec 2011 12:37:08 +0200
Subject: [PATCH 2/3] Install development headers

Add all public headers to install targets. This allows to build things
that depend on QtQuick3D components.
---
 src/quick3d/quick3d.pro |    5 +++++
 src/threed/threed.pro   |    5 +++++
 2 files changed, 10 insertions(+), 0 deletions(-)

diff --git a/src/quick3d/quick3d.pro b/src/quick3d/quick3d.pro
index cacb165..e3d4de8 100644
--- a/src/quick3d/quick3d.pro
+++ b/src/quick3d/quick3d.pro
@@ -35,10 +35,7 @@ DEFINES += QT_BUILD_QT3D_QUICK_LIB
 
 !contains(QT_CONFIG, egl):DEFINES += QT_NO_EGL
 
-devel.files = $$PUBLIC_HEADERS
-devel.path = $$[QT_INSTALL_HEADERS]/$$TARGET
 
-INSTALLS += devel
diff --git a/src/threed/threed.pro b/src/threed/threed.pro
index cbc88ef..7e29900 100644
--- a/src/threed/threed.pro
+++ b/src/threed/threed.pro
@@ -24,3 +24,8 @@ HEADERS += $$PRIVATE_HEADERS
 DEFINES += QT_BUILD_QT3D_LIB
 
 !contains(QT_CONFIG, egl):DEFINES += QT_NO_EGL
+
+devel.files = $$PUBLIC_HEADERS
+devel.path = $$[QT_INSTALL_HEADERS]/$$TARGET
+
+INSTALLS += devel
-- 
1.7.7.3

